client/dashboard/components/flash-message/index.tsx

```diff
@@ -14,7 +14,7 @@ interface FlashMessageProps {
  * Clears the query parameter when done.
  */
 export default function FlashMessage( {
-	key = 'updated',
+	param = 'updated',
```

Comment:
Thanks for catching. This seems fine. Maybe `paramName`? I was trying to choose names that avoided the confusion between which prop was referring to the query param value and which was referring to the query param name. And `param` _might_ be interpreted as the param's value 🤷‍♂️